python/pyspark/tests.py

```diff
@@ -1515,12 +1515,12 @@ def test_oldhadoop(self):
 
         conf = {
             "mapred.output.format.class": "org.apache.hadoop.mapred.SequenceFileOutputFormat",
```

Comment:
I'm not sure what this key was supposed to be before; maybe `mapreduce.outputformat.class`? but it can be `mapreduce.job.outputformat.class` now?